doc/whats_new.rst

```diff
@@ -134,6 +134,8 @@ Changelog
 
 Bug
 ~~~
+- Fix `mne.viz.utils._compute_scalings` to do channel-wise autoscale by `Stefan Repplinger`_
```

Comment:
Don't put private functions in `whats_new.rst`, instead document what public function / params the user would use to hit this ilke
```
- Fix :meth:`mne.Epochs.plot` with ``scalings='auto'`` to properly compute channel-wise scalings by `Stefan Repplinger`_
```

Reply:
Thank you